{"uid":"5105327d2bd8db40","name":"Ошибка при создании борды с None вместо списков в полях groups/typesList/customFields","fullName":"tests.test_backend.board.test_board#test_create_board_with_none_fields","historyId":"82caea6867a69708f62244f113e9e77f","time":{"start":1749798755156,"stop":1749798755421,"duration":265},"status":"passed","flaky":false,"newFailed":false,"newBroken":false,"newPassed":true,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"delete_output_dir","time":{"start":1749798742635,"stop":1749798742635,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"owner_client","time":{"start":1749798742635,"stop":1749798743126,"duration":491},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"base_url","time":{"start":1749798742635,"stop":1749798742635,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"pytestconfig","time":{"start":1749798742635,"stop":1749798742635,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"_verify_url","time":{"start":1749798742635,"stop":1749798742635,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"temp_space","time":{"start":1749798743126,"stop":1749798744123,"duration":997},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"temp_project","time":{"start":1749798744123,"stop":1749798745984,"duration":1861},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"testStage":{"status":"passed","steps":[{"name":"Отправка запроса, где списочные поля переданы как None","time":{"start":1749798755157,"stop":1749798755421,"duration":264},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"Проверка, что API вернул 400 – ошибка валидации типов","time":{"start":1749798755421,"stop":1749798755421,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":2,"attachmentsCount":0,"hasContent":true},"afterStages":[{"name":"temp_space::0","time":{"start":1749798778176,"stop":1749798778980,"duration":804},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"labels":[{"name":"tag","value":"backend"},{"name":"parentSuite","value":"tests.test_backend.board"},{"name":"suite","value":"test_board"},{"name":"host","value":"pkrvmxyh4eaekms"},{"name":"thread","value":"2537-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_backend.board.test_board"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"history":{"statistic":{"failed":0,"broken":8,"skipped":0,"passed":6,"unknown":0,"total":14},"items":[{"uid":"796a34c7d7127668","reportUrl":"https://vaizcom.github.io/autotests/128//#testresult/796a34c7d7127668","status":"broken","statusDetails":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.vaiz.dev', port=443): Max retries exceeded with url: /v4/Login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f254b41e350>, 'Connection to api.vaiz.dev timed out. (connect timeout=None)'))","time":{"start":1749798177025,"stop":1749798177025,"duration":0}},{"uid":"af640c9bf9c627d0","reportUrl":"https://vaizcom.github.io/autotests/127//#testresult/af640c9bf9c627d0","status":"passed","time":{"start":1749797903686,"stop":1749797903949,"duration":263}},{"uid":"3f67bb6888d3cbec","reportUrl":"https://vaizcom.github.io/autotests/126//#testresult/3f67bb6888d3cbec","status":"passed","time":{"start":1749745013576,"stop":1749745013829,"duration":253}},{"uid":"4cdae799b45a4bb5","reportUrl":"https://vaizcom.github.io/autotests/125//#testresult/4cdae799b45a4bb5","status":"passed","time":{"start":1749744413729,"stop":1749744414007,"duration":278}},{"uid":"ff77f297837694c0","reportUrl":"https://vaizcom.github.io/autotests/124//#testresult/ff77f297837694c0","status":"passed","time":{"start":1749744326292,"stop":1749744326608,"duration":316}},{"uid":"98b0733d30ea9694","reportUrl":"https://vaizcom.github.io/autotests/123//#testresult/98b0733d30ea9694","status":"broken","statusDetails":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.vaiz.dev', port=443): Max retries exceeded with url: /v4/Login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f6ed7455e50>, 'Connection to api.vaiz.dev timed out. (connect timeout=None)'))","time":{"start":1749744004615,"stop":1749744004615,"duration":0}},{"uid":"123589d005a11b3c","reportUrl":"https://vaizcom.github.io/autotests/122//#testresult/123589d005a11b3c","status":"passed","time":{"start":1749743629985,"stop":1749743630249,"duration":264}},{"uid":"89c33e15ce5a4966","reportUrl":"https://vaizcom.github.io/autotests/119//#testresult/89c33e15ce5a4966","status":"broken","statusDetails":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.vaiz.dev', port=443): Max retries exceeded with url: /v4/Login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fc7b5913490>, 'Connection to api.vaiz.dev timed out. (connect timeout=None)'))","time":{"start":1749742821714,"stop":1749742821714,"duration":0}},{"uid":"ed5ebb92414b2266","reportUrl":"https://vaizcom.github.io/autotests/118//#testresult/ed5ebb92414b2266","status":"broken","statusDetails":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.vaiz.dev', port=443): Max retries exceeded with url: /v4/Login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f9cb88f1f10>, 'Connection to api.vaiz.dev timed out. (connect timeout=None)'))","time":{"start":1749742669157,"stop":1749742669157,"duration":0}},{"uid":"3dbe509aff7368eb","reportUrl":"https://vaizcom.github.io/autotests/117//#testresult/3dbe509aff7368eb","status":"broken","statusDetails":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.vaiz.dev', port=443): Max retries exceeded with url: /v4/Login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f2b9f21f250>, 'Connection to api.vaiz.dev timed out. (connect timeout=None)'))","time":{"start":1749742498963,"stop":1749742498963,"duration":0}},{"uid":"c16efb51ba7be7d7","reportUrl":"https://vaizcom.github.io/autotests/116//#testresult/c16efb51ba7be7d7","status":"broken","statusDetails":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.vaiz.dev', port=443): Max retries exceeded with url: /v4/Login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7efcc7f78ad0>, 'Connection to api.vaiz.dev timed out. (connect timeout=None)'))","time":{"start":1749741235330,"stop":1749741235330,"duration":0}},{"uid":"811bdbc665af33b","reportUrl":"https://vaizcom.github.io/autotests/115//#testresult/811bdbc665af33b","status":"broken","statusDetails":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.vaiz.dev', port=443): Max retries exceeded with url: /v4/Login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f28d1ddcd10>, 'Connection to api.vaiz.dev timed out. (connect timeout=None)'))","time":{"start":1749740827910,"stop":1749740827910,"duration":0}},{"uid":"41a6bad377230761","reportUrl":"https://vaizcom.github.io/autotests/114//#testresult/41a6bad377230761","status":"broken","statusDetails":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.vaiz.dev', port=443): Max retries exceeded with url: /v4/Login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f271b110ad0>, 'Connection to api.vaiz.dev timed out. (connect timeout=None)'))","time":{"start":1749740654587,"stop":1749740654587,"duration":0}}]},"tags":["backend"]},"source":"5105327d2bd8db40.json","parameterValues":[]}